docs/pages/includes/install-homebrew.mdx

```diff
@@ -0,0 +1,85 @@
+  <Admonition type="note" scope={["enterprise", "cloud"]} scopeOnly>
```

Comment:
I think we should make the homebrew section much smaller since we don't officially support it, it's not released along with our other artifacts, and it doesn't support a number of features.

A small note that says "there is a community maintained formula but we recommend using the official releases" should be sufficient IMO.

Reply:
Such a note exists below:

>  The Teleport package in Homebrew is not maintained by Teleport and we can't
>  guarantee its reliability or security. We recommend the use of our [official
>  Teleport packages](https://goteleport.com/teleport/download?os=mac).

I'll consolidate while editing